docs/cloud-native-security/d4c-get-started.asciidoc

```diff
@@ -43,23 +43,28 @@ One of the <<d4c-default-policies, default D4C policies>> sends process telemetr
 
 In order to detect threats using this data, you'll need active <<detection-engine-overview, detection rules>>. Elastic has prebuilt detection rules designed for this data. (You can also create your own <<rules-ui-create, custom rules>>.)
 
-To install and enable Elastic's prebuilt rules:
+To install and enable the prebuilt rules:
 
 . Go to *Security > Manage > Rules*, then click *Add Elastic rules*.
-. Use the *Tags* selector to search for `container`. Select the `Data Source: Elastic Defend for Containers` tag. The rules table displays relevant rules.
-. Select all the displayed rules and click *Install _x_ selected rule(s)*.
-. Once the rules have loaded, you will see the Rules management page. Use the *Tags* selector to search for `container`. Select the `Container Workload Protection` tag.
+. Use the *Tags* selector to search for `container`. Select the `Data Source: Elastic Defend for Containers` tag.
+. Select all the displayed rules, then click *Install _x_ selected rule(s)*.
+. Return to the Rules page. Use the *Tags* selector to search for `container`. Select the `Data Source: Elastic Defend for Containers` tag.
 . Select all the rules with the tag, and then click *Bulk actions > Enable*.
 
+
 [[d4c-get-started-drift]]
 [discrete]
 == Get started with drift detection and prevention
 
-{elastic-sec} defines container drift as creating a new executable or modifying an existing executable within a container. Blocking drift restricts the number of attack vectors available to bad actors by prohibiting them from using external tools.
+{elastic-sec} defines container drift as the creation or modification of an executable within a container. Blocking drift restricts the number of attack vectors available to bad actors by prohibiting them from using external tools.
+
+To enable drift detection, you can use the default D4C policy:
 
-One of the <<d4c-default-policies, default D4C policies>> creates alerts in {elastic-sec} when container drift is detected. Before you enable blocking, we strongly recommend that you observe a production workload using the default policy to ensure that the workload does not create or modify executables as part of its normal operation.
+. Make sure the <<d4c-default-policies, default policy>> has not been deleted and remains active.
+. Make sure you enabled at least the "Container Workload Protection" rule, by following the steps to install prebuilt rules, above.
```

Comment:
This also looks like a pre-req to enabling drift detection, not the actual steps for doing it.